base/spec/lib/stellar/operation_spec.rb

```diff
@@ -296,3 +296,37 @@
     end
   end
 end
+
+RSpec.describe Stellar::Operation, ".manage_buy_offer" do
+  let(:buying_issuer) { Stellar::KeyPair.from_address("GDGU5OAPHNPU5UCLE5RDJHG7PXZFQYWKCFOEXSXNMR6KRQRI5T6XXCD7") }
+  let(:buying_asset) { Stellar::Asset.alphanum4("USD", buying_issuer) }
+
+  let(:selling_issuer) { Stellar::KeyPair.master }
+  let(:selling_asset) { Stellar::Asset.alphanum4("EUR", selling_issuer) }
+
+  it "creates a ManageBuyOfferOp" do
+    op = Stellar::Operation.manage_buy_offer(buying: buying_asset, selling: selling_asset, buyAmount: 50, price: 10)
+    expect(op.body.value).to be_an_instance_of(Stellar::ManageBuyOfferOp)
+    expect(op.body.value.buying).to eq(Stellar::Asset.alphanum4("USD", buying_issuer))
+    expect(op.body.value.selling).to eq(Stellar::Asset.alphanum4("EUR", selling_issuer))
+    expect(op.body.value.buyAmount).to eq(50)
```

Comment:
```suggestion
    expect(op.body.value.buy_amount).to eq(500000000)
```

This is "raw" xdr value, so it will be in stroops, not in lumens. Also in Ruby we use snake case for field names.